src/utils/index.js

```diff
@@ -38,20 +45,22 @@ export const renderPageLinks = (currentPage, numberOfPages) => {
 
 // call this function in 'componentDidMount' of any 'page component' (that renders paginated data) and pass 'this' as the argument
 export const setCurrentPage = (component) => {
-  const urlSearchParams = new URLSearchParams(location.search);
-  let page = parseInt(urlSearchParams.get('page'));
-  if (isNaN(page)) page = 1;
+  const urlSearchParams = new URLSearchParams(window.location.search);
+  let page = parseInt(urlSearchParams.get('page'), 10);
+  if (Number.isNaN(page)) page = 1;
   component.setState({ currentPage: page });
 };
 
 export const filterArticlesByLikes = (articles) => {
   const topArticles = articles
-    .sort((a, b) => (a.likes > b.likes) ? 1 : ((b.likes < a.likes) ? -1 : 0)).slice(0, 7);
+    .sort((a, b) => (a.likes > b.likes ? 1 : b.likes < a.likes ? -1 : 0))
```

Comment:
Let's at least correct the logic here before we create another function to refactor it. This should be:
```suggestion
    .sort((a, b) => (a.likes > b.likes ? 1 : a.likes < b.likes ? -1 : 0))
```

Reply:
I didn't touch this. This is Tiku's work. I think the parenthesis were removed by my code formatter on save (I worked on some other functions in this file, but not this).

Reply:
Errrmmm... if something changed within your PR who is responsible for the change but you? 🤷🏾‍♂️
Meanwhile, you missed the actual point. Take a closer look at the changes suggested in my comment.

Reply:
Okay